service/src/main/resources/config/verily/sdd/expanded/entity/platform_occurrence.json

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
I see expanded files for `platform_occurrence` entity and `platform_person_occurrence` entity group, but no original files. Did you want to go with the `GROUP_ITEMS` or `CRITERIA_OCCURRENCE` entity group type? You can delete the expanded files that you don't need anymore.